### ISSUE TITLE
Update version to [1.22] for [gitea] 10￥

### ISSUE THREAD

**Issue by qiaofeng1227** [Websoft9/docker-library#781]
**What type of your feature request?**  

Change the W9_VERSION to target version number at `.env` and test it

**Research and development steps your need to do**  

- [x] Check the official installation docs of this application
- [x] Add a comment with docker logs
- [x] If need to update documentation, add issue at [Websoft9 docs](https://github.com/websoft9/doc.websoft9.com)

**How to commit your change to [docker-library](https://github.com/Websoft9/docker-library)?**   

   ```
   docker exec -it websoft9-apphub bash
   apphub subimt --appid --github_token
   ```

**Testing steps**   

- [x] Upload your sreenshoot to PR
- [ ] Automation testing when submit PR to dev branch
- [x] System testing for dev branch

**Comment by cbnb233** [comment]
![image](https://github.com/user-attachments/assets/bf306cd3-3f54-4172-bc35-b5676c903c61)

**Comment by cbnb233** [comment]
docker logs gitea_g5w81
```
Generating /data/ssh/ssh_host_ed25519_key...
Generating /data/ssh/ssh_host_rsa_key...
2024/10/31 14:13:46 cmd/web.go:242:runWeb() [I] Starting Gitea on PID: 15
2024/10/31 14:13:46 cmd/web.go:111:showWebStartupMessage() [I] Gitea version: 1.22.3 built with GNU Make 4.4.1, go1.22.8 : bindata, timetzdata, sqlite, sqlite_unlock_notify
2024/10/31 14:13:46 cmd/web.go:112:showWebStartupMessage() [I] * RunMode: prod
2024/10/31 14:13:46 cmd/web.go:113:showWebStartupMessage() [I] * AppPath: /usr/local/bin/gitea
2024/10/31 14:13:46 cmd/web.go:114:showWebStartupMessage() [I] * WorkPath: /data/gitea
2024/10/31 14:13:46 cmd/web.go:115:showWebStartupMessage() [I] * CustomPath: /data/gitea
2024/10/31 14:13:46 cmd/web.go:116:showWebStartupMessage() [I] * ConfigFile: /data/gitea/conf/app.ini
2024/10/31 14:13:46 cmd/web.go:117:showWebStartupMessage() [I] Prepare to run install page
Generating /data/ssh/ssh_host_ecdsa_key...
Server listening on :: port 222.
Server listening on 0.0.0.0 port 222.
2024/10/31 14:13:46 cmd/web.go:304:listen() [I] Listen: http://0.0.0.0:3000
2024/10/31 14:13:46 cmd/web.go:308:listen() [I] AppURL(ROOT_URL): http://localhost:3000/
2024/10/31 14:13:46 ...s/graceful/server.go:50:NewServer() [I] Starting new Web server: tcp:0.0.0.0:3000 on PID: 15
2024/10/31 14:15:32 ...s/install/install.go:515:SubmitInstall() [I] Save settings to custom config file /data/gitea/conf/app.ini
2024/10/31 14:15:32 routers/common/db.go:23:InitDBEngine() [I] Beginning ORM engine initialization.
2024/10/31 14:15:32 routers/common/db.go:30:InitDBEngine() [I] ORM engine initialization attempt #1/10...
2024/10/31 14:15:32 ...s/install/install.go:594:SubmitInstall() [I] First-time run install finished!
2024/10/31 14:15:35 cmd/web.go:355:listen() [I] HTTP Listener: 0.0.0.0:3000 Closed
2024/10/31 14:15:35 cmd/web.go:111:showWebStartupMessage() [I] Gitea version: 1.22.3 built with GNU Make 4.4.1, go1.22.8 : bindata, timetzdata, sqlite, sqlite_unlock_notify
2024/10/31 14:15:35 cmd/web.go:112:showWebStartupMessage() [I] * RunMode: prod
2024/10/31 14:15:35 cmd/web.go:113:showWebStartupMessage() [I] * AppPath: /usr/local/bin/gitea
2024/10/31 14:15:35 cmd/web.go:114:showWebStartupMessage() [I] * WorkPath: /data/gitea
2024/10/31 14:15:35 cmd/web.go:115:showWebStartupMessage() [I] * CustomPath: /data/gitea
2024/10/31 14:15:35 cmd/web.go:116:showWebStartupMessage() [I] * ConfigFile: /data/gitea/conf/app.ini
2024/10/31 14:15:35 cmd/web.go:117:showWebStartupMessage() [I] Prepare to run web server
2024/10/31 14:15:35 routers/init.go:116:InitWebInstalled() [I] Git version: 2.45.2 (home: /data/gitea/home)
2024/10/31 14:15:36 ...s/setting/session.go:77:loadSessionFrom() [I] Session Service Enabled
2024/10/31 14:15:36 ...s/storage/storage.go:176:initAttachments() [I] Initialising Attachment storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/attachments
2024/10/31 14:15:36 ...s/storage/storage.go:166:initAvatars() [I] Initialising Avatar storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/avatars
2024/10/31 14:15:36 ...s/storage/storage.go:192:initRepoAvatars() [I] Initialising Repository Avatar storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/repo-avatars
2024/10/31 14:15:36 ...s/storage/storage.go:186:initLFS() [I] Initialising LFS storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/git/lfs
2024/10/31 14:15:36 ...s/storage/storage.go:198:initRepoArchives() [I] Initialising Repository Archive storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/repo-archive
2024/10/31 14:15:36 ...s/storage/storage.go:208:initPackages() [I] Initialising Packages storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/packages
2024/10/31 14:15:36 ...s/storage/storage.go:219:initActions() [I] Initialising Actions storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/actions_log
2024/10/31 14:15:36 ...s/storage/storage.go:223:initActions() [I] Initialising ActionsArtifacts storage with type: local
2024/10/31 14:15:36 ...les/storage/local.go:33:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/actions_artifacts
2024/10/31 14:15:36 routers/init.go:138:InitWebInstalled() [I] SQLite3 support is enabled
2024/10/31 14:15:36 routers/common/db.go:23:InitDBEngine() [I] Beginning ORM engine initialization.
2024/10/31 14:15:36 routers/common/db.go:30:InitDBEngine() [I] ORM engine initialization attempt #1/10...
2024/10/31 14:15:36 cmd/web.go:194:serveInstalled() [I] PING DATABASE sqlite3
2024/10/31 14:15:36 cmd/web.go:194:serveInstalled() [W] Table system_setting Column version db default is , struct default is 1
2024/10/31 14:15:36 routers/init.go:144:InitWebInstalled() [I] ORM engine initialization successful!
2024/10/31 14:15:38 routers/init.go:85:syncAppConfForGit() [I] AppPath changed from '' to '/usr/local/bin/gitea'
2024/10/31 14:15:38 routers/init.go:90:syncAppConfForGit() [I] CustomConf changed from '' to '/data/gitea/conf/app.ini'
2024/10/31 14:15:38 routers/init.go:96:syncAppConfForGit() [I] re-sync repository hooks ...
2024/10/31 14:15:38 routers/init.go:99:syncAppConfForGit() [I] re-write ssh public keys ...
2024/10/31 14:15:38 ...er/issues/indexer.go:76:func1() [I] PID 15: Initializing Issue Indexer: bleve
2024/10/31 14:15:38 ...xer/stats/indexer.go:41:populateRepoIndexer() [I] Populating the repo stats indexer with existing repositories
2024/10/31 14:15:38 ...er/issues/indexer.go:153:func2() [I] Issue Indexer Initialization took 13.031423ms
2024/10/31 14:15:38 ...s/actions/cleanup.go:36:cleanExpiredArtifacts() [I] Found 0 expired artifacts
2024/10/31 14:15:38 ...s/actions/cleanup.go:60:cleanNeedDeleteArtifacts() [I] Found 0 artifacts pending deletion
2024/10/31 14:15:38 cmd/web.go:304:listen() [I] Listen: http://0.0.0.0:3000
2024/10/31 14:15:38 cmd/web.go:308:listen() [I] AppURL(ROOT_URL): http://gitea.test2.websoft9.cn/
2024/10/31 14:15:38 cmd/web.go:311:listen() [I] LFS server enabled
2024/10/31 14:15:38 ...s/graceful/server.go:50:NewServer() [I] Starting new Web server: tcp:0.0.0.0:3000 on PID: 15
2024/10/31 14:15:39 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/login for 175.8.127.187:0, 200 OK in 2.3ms @ auth/auth.go:164(auth.SignIn)
2024/10/31 14:15:39 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/login for 175.8.127.187:0, 200 OK in 1.5ms @ auth/auth.go:164(auth.SignIn)
2024/10/31 14:16:27 ...ers/web/auth/auth.go:231:SignInPost() [I] Failed authentication attempt for admin from 175.8.127.187:0: user does not exist [uid: 0, name: admin]
2024/10/31 14:16:27 ...eb/routing/logger.go:102:func1() [I] router: completed POST /user/login for 175.8.127.187:0, 200 OK in 9.5ms @ auth/auth.go:196(auth.SignInPost)
2024/10/31 14:16:31 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/sign_up for 175.8.127.187:0, 200 OK in 2.0ms @ auth/auth.go:425(auth.SignUp)
2024/10/31 14:16:39 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/login for 175.8.127.187:0, 200 OK in 1.7ms @ auth/auth.go:164(auth.SignIn)
2024/10/31 14:16:44 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/sign_up for 175.8.127.187:0, 200 OK in 1.8ms @ auth/auth.go:425(auth.SignUp)
2024/10/31 14:16:45 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/login/openid for 175.8.127.187:0, 200 OK in 1.5ms @ auth/openid.go:30(auth.SignInOpenID)
2024/10/31 14:16:50 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/sign_up for 175.8.127.187:0, 200 OK in 1.4ms @ auth/auth.go:425(auth.SignUp)
Connection closed by 185.242.226.53 port 45426 [preauth]
2024/10/31 14:18:21 ...eb/routing/logger.go:102:func1() [I] router: completed POST /user/sign_up for 175.8.127.187:0, 200 OK in 1.5ms @ auth/auth.go:453(auth.SignUpPost)
2024/10/31 14:18:29 ...eb/routing/logger.go:102:func1() [I] router: completed POST /user/sign_up for 175.8.127.187:0, 303 See Other in 115.6ms @ auth/auth.go:453(auth.SignUpPost)
2024/10/31 14:18:29 ...odels/user/avatar.go:55:GenerateRandomAvatar() [I] New random avatar created: 1
2024/10/31 14:18:29 ...eb/routing/logger.go:102:func1() [I] router: completed GET / for 175.8.127.187:0, 200 OK in 28.3ms @ web/home.go:32(web.Home)
2024/10/31 14:18:29 ...eb/routing/logger.go:102:func1() [I] router: completed GET /avatars/99b3068b5857c780d408c5e937d6efe0?size=48 for 175.8.127.187:0, 200 OK in 0.3ms @ web/base.go:22(avatars)
2024/10/31 14:18:30 ...eb/routing/logger.go:102:func1() [I] router: completed GET /repo/search?count_only=1&uid=1&team_id=undefined&q=&page=1&mode= for 175.8.127.187:0, 200 OK in 1.5ms @ repo/repo.go:556(repo.SearchRepo)
2024/10/31 14:18:30 ...eb/routing/logger.go:102:func1() [I] router: completed GET /repo/search?sort=updated&order=desc&uid=1&team_id=undefined&q=&page=1&limit=15&mode=&archived=false for 175.8.127.187:0, 200 OK in 3.5ms @ repo/repo.go:556(repo.SearchRepo)
2024/10/31 14:18:33 ...eb/routing/logger.go:68:func1() [I] router: polling   GET /user/events for 175.8.127.187:0, elapsed 3328.1ms @ events/events.go:18(events.Events)
2024/10/31 14:19:06 ...eb/routing/logger.go:102:func1() [I] router: completed GET /user/events for 175.8.127.187:0, 200 OK in 36439.8ms @ events/events.go:18(events.Events)
```
